opencv/src/main/java/org/bytedeco/javacpp/cvkernels.java

```diff
@@ -1,98 +0,0 @@
-/*
- * Copyright (C) 2009-2012 Samuel Audet
- *
- * Licensed either under the Apache License, Version 2.0, or (at your option)
- * under the terms of the GNU General Public License as published by
- * the Free Software Foundation (subject to the "Classpath" exception),
- * either version 2, or any later version (collectively, the "License");
- * you may not use this file except in compliance with the License.
- * You may obtain a copy of the License at
- *
- *     http://www.apache.org/licenses/LICENSE-2.0
- *     http://www.gnu.org/licenses/
- *     http://www.gnu.org/software/classpath/license.html
- *
- * or as provided in the LICENSE.txt file that accompanied this code.
- * Unless required by applicable law or agreed to in writing, software
- * distributed under the License is distributed on an "AS IS" BASIS,
- * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
- * See the License for the specific language governing permissions and
- * limitations under the License.
- */
-
-package org.bytedeco.javacpp;
-
-import java.nio.DoubleBuffer;
-import java.util.Arrays;
-import org.bytedeco.javacpp.annotation.ByRef;
-import org.bytedeco.javacpp.annotation.MemberSetter;
-import org.bytedeco.javacpp.annotation.Name;
-import org.bytedeco.javacpp.annotation.Platform;
-import org.bytedeco.javacpp.annotation.Properties;
-
-import static org.bytedeco.javacpp.Loader.*;
-import static org.bytedeco.javacpp.opencv_core.*;
-
-/**
- *
- * @author Samuel Audet
- */
-@Properties(inherit=opencv_core.class, value={
-    @Platform(define={"MAX_SIZE 16", "CV_INLINE static inline"}, include="cvkernels.h", compiler="fastfpu") })
-public class cvkernels {
```

Comment:
We'll need to add back this file as well. It's a few custom operations that I've added before that are used in JavaCV...